philip/src/Router.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
+  import { onDestroy } from "svelte";
+
+  export let routes;
+  export let curRoute;
+
+  export let updateSession;
+  export let active_tab;
+  export let session;
+
+  let component;
+  const unsubscribe = curRoute.subscribe(value => {
+    console.log("route", $curRoute, active_tab);
+    component = routes[$curRoute].component;
+  });
+  onDestroy(unsubscribe);
+</script>
+
+<style>
+
+</style>
+
+<div class="content">
+  <svelte:component
+    this={component}
+    {active_tab}
```

Comment:
Same timeline loads issue
Or maybe we could use curRoute here instead of active_tab